pandas/core/tools/timedeltas.py

```diff
@@ -23,8 +29,39 @@
 
 from pandas.core.arrays.timedeltas import sequence_to_td64ns
 
+if TYPE_CHECKING:
+    from pandas._libs.tslibs.timedeltas import UnitChoices
+    from pandas._typing import (
+        AnyArrayLike,
+        DateTimeErrorChoices,
+    )
+
+    from pandas import Series
+
+
+@overload
+def to_timedelta(
+    arg: str | int | float | timedelta,
+    unit: UnitChoices | None = ...,
+    errors: DateTimeErrorChoices = ...,
+) -> Timedelta:
+    ...
+
+
+@overload
+def to_timedelta(
+    arg: list | tuple | range | AnyArrayLike,
+    unit: UnitChoices | None = ...,
+    errors: DateTimeErrorChoices = ...,
+) -> Series:
+    ...
+
 
-def to_timedelta(arg, unit=None, errors="raise"):
+def to_timedelta(
+    arg: str | int | float | timedelta | list | tuple | range | AnyArrayLike,
+    unit: UnitChoices | None = None,
+    errors: DateTimeErrorChoices = "raise",
+) -> Timedelta | Series:
```

Comment:
Do list-like `arg` return a `TimedeltaIndex`?
![image](https://user-images.githubusercontent.com/6618166/158037570-e4ae62ae-3dc3-49d8-aeee-268776a4dd35.png)
Might need to update the documentation if that isn't the case anymore.

Reply:
good catch.  Fixed in https://github.com/pandas-dev/pandas/pull/46343/commits/7b31def4f2e191cdbd98b1346e5298756c0e1671